src/main/java/seedu/address/logic/commands/consultation/AddToConsultCommand.java

```diff
@@ -33,27 +36,40 @@ public class AddToConsultCommand extends Command {
             + "Example: " + COMMAND_WORD + " 1 n/John Doe n/Harry Ng";
 
     public static final String MESSAGE_ADD_TO_CONSULT_SUCCESS = "Added students to the Consultation: %1$s";
-    public static final String MESSAGE_DUPLICATE_STUDENT_IN_CONSULTATION = "%s is already added to the consultation!";
+
+    public static final String MESSAGE_STUDENT_NOT_FOUND = "Student(s) %s not found";
+    public static final String MESSAGE_DUPLICATE_STUDENT_IN_CONSULTATION_BY_NAME =
+            "%s is already added to the consultation!";
+    public static final String MESSAGE_DUPLICATE_STUDENT_IN_CONSULTATION_BY_INDEX =
```

Comment:
I like the distinction between the name and the index error message. Helps the user determine which part of their command is invalid.